packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -117,21 +120,25 @@ def _create_all_clients(directory_out: Optional[str]) -> None:
     validate_measurement_service_classes(measurement_service_classes)
 
     for service_class in measurement_service_classes:
-        base_service_class = extract_base_service_class(service_class)
-        module_name = create_module_name(base_service_class, generated_modules)
-        class_name = create_class_name(base_service_class)
-        validate_identifier(module_name, "module")
-        validate_identifier(class_name, "class")
-
-        _create_client(
-            channel_pool=channel_pool,
-            discovery_client=discovery_client,
-            measurement_service_class=service_class,
-            module_name=module_name,
-            class_name=class_name,
-            directory_out=directory_out_path,
-        )
-        generated_modules.append(module_name)
+        try:
+            base_service_class = extract_base_service_class(service_class)
+            module_name = create_module_name(base_service_class, generated_modules)
+            class_name = create_class_name(base_service_class)
+            validate_identifier(module_name, "module")
+            validate_identifier(class_name, "class")
+
+            _create_client(
+                channel_pool=channel_pool,
+                discovery_client=discovery_client,
+                measurement_service_class=service_class,
+                module_name=module_name,
+                class_name=class_name,
+                directory_out=directory_out_path,
+            )
+            generated_modules.append(module_name)
+        except Exception as e:
+            click.echo(CLIENT_CREATION_ERROR_MESSAGE.format(service_class, e))
+            continue
```

Comment:
nit: The `continue` statement skips the execution of the code block after the continue statement and forces the control to start the next iteration. So, having it at the end of the iteration has no effect.

Reply:
Thanks for pointing. I missed to remove it.
I have updated now.